cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -77,10 +80,12 @@ instance HasTextEnvelope UpdateProposal where
     textEnvelopeType _ = "UpdateProposalShelley"
 
 instance ToCBOR UpdateProposal where
-    toCBOR = toCBOR . toShelleyUpdate
+    toCBOR = toCBOR . toShelleyUpdate @StandardShelley
+    -- We have to pick a monomorphic era type for the serialisation. We use the
+    -- Shelley era. This makes no difference since era type is phantom.
```

Comment:
Can we define a type synonym to make this clear. Something like:

`type ShelleyOnwardsCrypto = StandardShelley` or something with a better name :grin: